cypress/support/commands.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="cypress" />
```

Comment:
Same here.

Reply:
I had removed that file, but it gave me error. So I had to put it back.